.github/workflows/companion-deploy.yml

```diff
@@ -4,8 +4,8 @@ name: Companion Deploy
 on:
   push:
     branches: [main]
```

Comment:
This will caust it to build on pushes to the main branch as well as tags. Is this what we want :thinking: ?

Reply:
Good catch!